src/index.html

```diff
@@ -11,7 +11,155 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset name="personalInformation" class="form-field">
+        <legend>Personal information</legend>
+
+        <label class="form-label">
+          Surname:
+          <input
+            type="text"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-label">
+          Name:
+          <input
+            type="text"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-label">
+            How old are You?
+          <input
+            type="number"
+            required
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-label">
+          Full date of birth:
+          <input type="date" required>
+        </label>
+
+        <label class="form-label">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset name="registration" class="form-field">
+        <legend>Registration</legend>
+
+        <label class="form-label">
+          E-mail:
+          <input type="email"
+          placeholder="email@example.com"
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="form-label">
+          Password:
+          <input
+            type="password"
+            minlength="5"
+            maxlength="15"
+            autocomplete="off"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset name="anInterestingFactAboutYou" class="form-field">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-label">
+          Do you love cats?
+          <label>
+            <input
+              name="catLover"
+              type="radio"
+              value="yes"
+            >
+              Yes
+          </label>
```

Comment:
same element level - same indentation
```suggestion
          <label>
            <input
              name="catLover"
              type="radio"
              value="yes"
            >
            Yes
          </label>
```